### PR TITLE
Add int pointer function

### LINKED ISSUE #214: Int pointer conversion is missing
Function `Int` for `int` pointer conversion is missing.

Example implementation in `pointer/pointer.go`:

```go
// Int returns a pointer to an int.
func Int(i int) *int {
	return &i
}
```

<sub>Tobias Giese <tobias.giese@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sub>

### PR DESCRIPTION
**Pull request by tobiasgiese**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #214

<sub>Tobias Giese <tobias.giese@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sub>

### CODE REVIEW

File: pointer/pointer.go [hunk 1]
```diff
@@ -46,6 +46,13 @@ func AllPtrFieldsNil(obj interface{}) bool {
 	return true
 }
 
+// Int returns a pointer to an int3
```
Comment (tobiasgiese) [inline pointer/pointer.go:49]:
```suggestion
// Int returns a pointer to an int
```

**Comment by tobiasgiese** [comment]
Cans omeone PTAL? 🙂 
/cc @apelisse @stewart-yu

**Review by thockin — COMMENTED** [review]
I think we want `IntDeref()` too for completeness?

**Comment by tobiasgiese** [comment]
> I think we want `IntDeref()` too for completeness?

You're totally right. I've pushed the changes.
Thanks for reviewing :)

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**